src/components/modal/modal.scss

```diff
@@ -184,13 +189,44 @@
  */
 .content {
   @apply relative box-border block h-full overflow-auto p-0;
-  max-block-size: 100%;
   background-color: var(--calcite-modal-content-background, theme("colors.background.foreground.1"));
-  padding-block: var(--calcite-modal-content-padding, var(--calcite-modal-padding-internal));
-  padding-inline: var(--calcite-modal-content-padding, var(--calcite-modal-padding-internal));
+  max-block-size: 100%;
+  padding: var(--calcite-modal-content-padding, var(--calcite-modal-padding-internal));
+}
+
+.content-top {
+  @apply bg-foreground-1
+  border-color-3
+  flex
+  min-w-0
+  max-w-full
+  rounded-t
+  border-0
+  border-b
+  border-solid
+  z-header;
+  flex: 0 0 auto;
+  padding: var(--calcite-modal-padding-internal);
+}
+
+.content-bottom {
+  @apply bg-foreground-1
+    border-color-3
+    mt-auto
+    box-border
+    flex
+    w-full
+    justify-between
+    rounded-b
+    border-0
+    border-t
+    border-solid
+    z-header;
+  flex: 0 0 auto;
+  padding: var(--calcite-modal-padding-internal);
 }
 
-.content--no-footer {
+.content--no-modal-footer {
```

Comment:
Since the slots were renamed, can this go back to the previous name?

This applies to other modal-footer renames in this PR.